less/mixins.less

```diff
@@ -1 +1,17 @@
-// Mixins Here
\ No newline at end of file
+.add-flex(@direction: row) {
+    display: flex;
+    flex-direction: @direction;
+}
+
+.flex-center {
+    justify-content: center;
+    align-items: center;
+}
+
+.signup-btn(@bg-color; @f-color) {
+    background-color: @bg-color;
+    color: @f-color;
+    font-size: 2rem;
+    border-radius: 5px;
+    padding: 10px 0;
+}
```

Comment:
- Navigation Styles: Use the navigation.less file for styling. - DONE!

- Main Content Styles: Use the home-page.less file for styling - DONE!

 - LESS Mixins: Create and use 2 different mixins to aid your styling. Use the mixins.less file for your mixins - DONE!

 - LESS Parametric Mixin: create a parametric mixin that is used to create the sign up button styles. - DONE!

- Use at least 2 parameters to create your button - DONE!

 - Create a hover state that swaps the background color and font color of the base button styles. - DONE!